### PR TITLE
Update cadvisor to 0.39.2

### PR DESCRIPTION
**Pull request by jweispfenning**

### CODE REVIEW

**Review by alexjplant — CHANGES_REQUESTED** [review]
```
docker pull k8s.gcr.io/cadvisor:v0.39.2
Error response from daemon: manifest for k8s.gcr.io/cadvisor:v0.39.2 not found: manifest unknown: Failed to fetch "v0.39.2" from request "/v2/cadvisor/manifests/v0.39.2".
```

I also **can't** pull the old image tag either. Perhaps Google changed where they're pushing it?

**Review by adamdecaf — APPROVED** [review]

File: lib/infra/17-cadvisor.yml [hunk 1]
```diff
@@ -86,7 +86,7 @@ spec:
     spec:
       automountServiceAccountToken: false
       containers:
-      - image: k8s.gcr.io/cadvisor:v0.30.2
+      - image: k8s.gcr.io/cadvisor:v0.39.2
```
Comment (adamdecaf) [inline lib/infra/17-cadvisor.yml:89]: Looks like they changed the URI to pull these images.

```
docker pull gcr.io/cadvisor/cadvisor:v0.39.2
```

https://console.cloud.google.com/gcr/images/cadvisor/GLOBAL/cadvisor?gcrImageListsize=30

Comment (jweispfenning) [reply lib/infra/17-cadvisor.yml]: interesting, looks like this was correct in modules but not here. will push up a change

Comment (jweispfenning) [reply lib/infra/17-cadvisor.yml]: @alexjplant FYI

**Review by alexjplant — APPROVED** [review]

**Pull request merged**